librocksdb_sys/crocksdb/c.cc

```diff
@@ -1944,6 +1948,27 @@ void crocksdb_options_destroy(crocksdb_options_t* options) {
   delete options;
 }
 
+void crocksdb_column_family_descriptor_destroy(
+    crocksdb_column_family_descriptor* cf_dec) {
+    delete cf_dec;
+}
+
+void crocksdb_list_column_family_descriptors_destroy(
+    crocksdb_column_family_descriptor** cf_decs, size_t cf_decs_len) {
+  for (size_t i = 0; i < cf_decs_len; ++i) {
+      crocksdb_column_family_descriptor_destroy(cf_decs[i]);
+  }
+  delete[] cf_decs;
+}
+
+void crocksdb_extract_column_family_descriptor(
+    const crocksdb_column_family_descriptor* const cf_dec, char** cf_name,
```

Comment:
Why two consts?

Reply:
This is a bit ugly, can we just do this in `crocksdb_load_latest_options` so we don't need to care about `ColumnFamilyDescriptor`?

Reply:
I think we really should expose ColumnFamilyDescriptor and let rocksdb.rs::load_latest_options take care of it, cause crocksdb_load_latest_options is better to staying the same with lower level